apps/client/src/common/components/view-params-editor/constants.ts

```diff
@@ -30,6 +30,14 @@ const hideTimerSeconds: ParamField = {
   defaultValue: false,
 };
 
+const showLeadingZeros: ParamField = {
```

Comment:
all other options are for hiding custom behaviour, should we keep it here as well? I guess we can just default it to true

Reply:
Honestly I don't know. Usually I prefer keeping the original behavior and use an option to change it.
User updating the app will probably be surprised that the default format of the timer has changed ?

But if you prefer i can reverse the condition to make it the new default

Reply:
if the variable was to hide, and the default was on, the users wouldnt see a difference.

My suggestion was with the inclination that all the other options are to hide things, so this one is different.

But I guess you may be correct in the idea that it doesnt matter if it is hide / show, what we are doing is changing the default behaviour. In which case your change makes more sense than my proposal. What do you think?

Reply:
Changing the text and the default value of the option should be nice to have all the option text to do a "hide function" but i struggle with the boolean parser here:
https://github.com/cpvalente/ontime/blob/master/apps/client/src/features/viewers/common/viewUtils.ts#L32-36

When the parameter is absent from the parameters part in url it will be parsed to "false" thus breaking the default value.
I don't know how to fix this so that's why I inverted the logic from "hide" to "show"